cluster-scope/base/operators.coreos.com/subscriptions/opf-grafana-public/subscription.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
The subscription here is the same as this one: 

https://github.com/operate-first/apps/tree/master/cluster-scope/base/operators.coreos.com/subscriptions/opf-grafana

I think we should rename `opf-grafana` to `grafana-operator` 

Then create a bundle for opf-grafana-public. In it, include: 

`base/core/namespaces/opf-grafana-public`
`base/operators.coreos.com/operatorgroups/opf-grafana-public`
`base/operators.coreos.com/subscriptions/grafana-operator`

Reply:
added the bundle :+1: